catboost/libs/metrics/metric.cpp

```diff
@@ -6387,6 +6471,12 @@ void CheckPreprocessedTarget(
                   "metric/loss-function " << lossFunction << " is a Multiclassification metric, "
                   " each target label should be a nonnegative integer");
     }
+
+    if (lossFunction == ELossFunction::MultiRMSE) {
```

Comment:
It will be better to write `if (lossFunction != ELossFunction::MultiRMSEWithMissingValues) {` here.

Reply:
ok.
Done